### DISCUSSION TITLE
Fusion can't connect to network on launch

### DISCUSSION THREAD

**Discussion by affirmativeconsequent** [cryinkfly/Autodesk-Fusion-360-for-Linux#301] [Q&A]
Hello, Fusion starts with the splash screen but then says there is no network and quits. Does anybody know what to do to fix this? I am fairly new at using Linux. I'm using Linux mint 21. Thank-you!

**Comment by mstixenberger** [comment]
Same here, unfortunately
any updates on this?

**Reply by affirmativeconsequent** [reply]
Are you also using Mint 21? I am curious if that is the problem, unfortunately I don't have time right now to find out. CAD is the only thing stopping me from leaving windows and I would love to get it figured out, its just too much of a time sink.
